test/test_skill.py

```
@@ -1,27 +1,22 @@
-import pytest
+from unittest.mock import Mock, patch
+from typing import Any, Generator
 from ovos_plugin_manager.skills import find_skill_plugins
 from ovos_utils.fakebus import FakeBus
 
 from skill_randomness import RandomnessSkill
 
 
 @pytest.fixture(scope="session")
-def test_skill(test_skill_id="skill-ovos-randomness.openvoiceos", bus=FakeBus()):
+def test_skill(test_skill_id="skill-ovos-randomness.openvoiceos", bus=FakeBus()) -> Generator[RandomnessSkill, Any, None]:
```

Ruff (0.8.2)

Do not perform function call `FakeBus` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)